client/lib/domains/whois/protected-contact-information.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * Internal dependencies
+ */
+import { isOpenHrsDomain, isOpenSrsDomain } from 'lib/domains';
+
+function getOpenHrsProtectedContactInformation( domain ) {
+	return {
+		firstName: 'Private',
+		lastName: 'Whois',
+		organization: 'Knock Knock Whois Not There LLC',
+		email: `${ domain }@privatewho.is`,
+		phone: '+1.8772738550',
+		address1: '60, 29th Street',
+		address2: '#343',
+		city: 'San Francisco',
+		state: 'CA',
+		postalCode: '94110-4929',
+		countryCode: 'US'
+	};
+}
+
+function getOpenSrsProtectedContactInformation( domain ) {
+	return {
+		firstName: '',
+		lastName: 'Contact Privacy Inc. Customer 012345',
+		organization: 'Contact Privacy Inc. Customer 012345',
+		email: `${ domain }@contactprivacy.com`,
+		phone: '+1.4165385457',
+		address1: '96',
+		address2: 'Mowat Ave',
+		city: 'Toronto',
+		state: 'ON',
+		postalCode: 'M6K 3M1',
+		countryCode: 'CA'
+	};
+}
+
+function getWwdProtectedContactInformation( domain ) {
```

Comment:
You make a good point here - I automatically also included WWD in this patch/approach. On the other hand, I remember the WHOIS information often lagging behind when disabling/enabling privacy protection (for example, during transfers). Which resulted in showing that the privacy is enabled but the preview of the public record showing private information (or vice versa). I can imagine some users behind confused/alarmed by this and contacting HEs as a result.
I think showing this "stable", hardcoded data is _overall_ better - if something goes wrong during enabling/disabling the privacy, the error will be shown on the frontend. If everything goes OK, we'll just have to trust the API that it did its job. Not ideal, but nor is the behaviour of the API :/